Arreglo en el rating de algunas consultas que se han cambiado

rojas1995 opened the pull request:
Aparte añado en el contex del rating la imagen de los user

Commit:
[fix] Correción de la comprobación de enroll

Commit:
[feat] Añado en el context el picture del user

Commit:
[refactor] Borro una variable que no se usa

angbongon reviewed (dismissed):
Todo correcto en principio

angbongon commented:
@angcallla revisa por la parte de QA

angelcalzadollamas reviewed (changes requested):
Cambios hablados por slack para que desaparezca la estrella si está en reject o pending el enrollment

Commit:
[fix] Boton puntuar

Ahora cuando tu solicitud no ha sido aceptada no aparece la estrella para puntuar

Commit:
[fix] Ahora sale el mensaje solo cuando es necesario
